README.rst

```diff
@@ -1,8 +1,7 @@
 openedx-completion-aggregator
 =============================
 
-|pypi-badge| |travis-badge| |codecov-badge| |doc-badge| |pyversions-badge|
-|license-badge|
+|pypi-badge| |travis-badge| |doc-badge| |pyversions-badge| |license-badge|
```

Comment:
License badge link needs to be updated to [`open-craft`](https://github.com/open-craft/openedx-completion-aggregator/blob/master/LICENSE.txt) too; [the README shows "repo not found"](https://github.com/open-craft/openedx-completion-aggregator/blob/5a883f8652f7eda81b9e22678ef0eab351846411/README.rst).